src/patchers/UnaryExistsOpPatcher.js

```diff
@@ -0,0 +1,24 @@
+import UnaryOpPatcher from './UnaryOpPatcher.js';
+
+export default class UnaryExistsOpPatcher extends UnaryOpPatcher {
+  patchAsExpression() {
+    const node = this.node;
+    const nodeExpression = node.expression;
+
+    // Only add parentheses if necessary
+    if (node.parentNode && node.parentNode.type !== 'Block') {
+        this.insertBefore('(');
+        this.insertAfter(')');
+    }
+
+    if (nodeExpression && nodeExpression.type === 'Identifier') {
```

Comment:
For extra credit, you should handle negation a la [`EqualityPatcher`](https://github.com/decaffeinate/decaffeinate/blob/master/src/patchers/EqualityPatcher.js). You should add tests like this:

```js
check(`
  unless a?
    b
`, `
  if (typeof a === 'undefined' || a === null) {
    b;
  }
`);

check(`
  unless a.b?
    b
`, `
  if (a.b == null) {
    b;
  }
`);
```